tests/lsp_features/hover.zig

```diff
@@ -830,6 +903,9 @@ test "type reference cycle" {
         \\    beta: @TypeOf(alpha),
         \\) void
         \\```
+        \\```zig
+        \\(unknown)
```

Comment:
I have another PR in the works: FnControlOption@8d5878bb (Will open after this one is merged)